CONTRIBUTING.md

```diff
@@ -336,4 +336,5 @@ Note: if you use VS Code as your editor, you can install the [prettier plugin][p
 [improve-exercise-metadata]: https://github.com/exercism/legacy-docs/blob/main/you-can-help/improve-exercise-metadata.md
 [legacy-docs]: https://github.com/exercism/legacy-docs
 [prettier]: https://prettier.io/
+[format-array]: https://github.com/exercism/problem-specifications/blob/main/bin/format-array.rb
```

Comment:
notice! this link does not work yet for the obvious reason that `format-array` doesn't exist on `main` yet! I am not sure if I have a better solution to offer for review, other than "you will just have to check this link very carefully"

Or, this change could be removed from this PR and only submitted in a separate PR after format-array.rb has been merged, so that all can verify that the link is correct, but I was not convinced that was a better solution.